### PR TITLE
Removed .SVG Icon and Added .PNG Icons

### PR DESCRIPTION
**Pull request by AHappyUser**
Removed .SVG icon file and added .PNG icons  sizes 16x16, 32x32, 64x64, 128x128, 256x256, and 512x512.

### CODE REVIEW

**Comment by TingPing** [comment]
I would have kept the svg and only added the 2 sizes appdata cares about personally.

File: io.exodus.Exodus.json [hunk 1]
```diff
@@ -22,7 +22,12 @@
       "build-commands": [
         "install -Dm644 io.exodus.Exodus.desktop /app/share/applications/exodus.desktop",
         "install -Dm644 io.exodus.Exodus.appdata.xml /app/share/appdata/io.exodus.Exodus.appdata.xml",
-        "install -Dm644 -t /app/share/icons/hicolor/scalable/apps io.exodus.Exodus.svg",
+        "install -Dm644 -t /app/share/icons/hicolor/16x16/apps io.exodus.Exodus-16x16.png",
+        "install -Dm644 -t /app/share/icons/hicolor/32x32/apps io.exodus.Exodus-32x32.png",
+        "install -Dm644 -t /app/share/icons/hicolor/64x64/apps io.exodus.Exodus-64x64.png",
+        "install -Dm644 -t /app/share/icons/hicolor/128x128/apps io.exodus.Exodus-128x128.png",
+        "install -Dm644 -t /app/share/icons/hicolor/256x256/apps io.exodus.Exodus-256x256.png",
+        "install -Dm644 -t /app/share/icons/hicolor/512x512/apps io.exodus.Exodus-512x512.png",
```
Comment (TingPing) [inline io.exodus.Exodus.json]: This fails because it has the wrong file name.

Comment (TingPing) [reply io.exodus.Exodus.json]: `install -Dm644 io.exodus.Exodus-512x512.png /app/share/icons/hicolor/512x512/apps/io.exodus.Exodus.png`

Comment (AHappyUser) [reply io.exodus.Exodus.json]: Just made a Pull Request and added 64x64 and 128x128 icons.

**Pull request closed without merging**